src/content/1.7/development/components/grid/tutorials/work-with-bulk-actions.md

```diff
@@ -1,6 +1,142 @@
 ---
-title: How to work with Bulk Actions?
+title: How to use Bulk actions in Grid?
 weight: 1
 ---
 
-> Do you know how to do it? Please feel free to contribute to the docs!
\ No newline at end of file
+# How to use Bulk actions in Grid?
+
+In addition to displaying data tables, Grid component also comes with a lot of additional functionality. One of it is Bulk actions.
+
+## What is Bulk action?
+
+Bulk action is action (e.g. Delete, Change status & etc) that can be performed on multiple entries in table. This is a common task when using Grid (e.g. Change status for multiple Orders in single action).
```

Comment:
```suggestion
Bulk action is action (e.g. Delete, Change status & etc) that can be performed on multiple entries in table. This is a common task when using Grid (e.g. Delete multiple selected Orders with a single click).
```
I think this example is really widely known and used 🤔

Reply:
agree, commited :)

Reply:
I can't accept your change as `Delete` action does not exist for Order in PrestaShop. :smiling_imp:

Reply:
Aaaargh indeed 😅. "Delete bulk suppliers" then ? I thought that people often need to delete things in batches (like after having creating multiple dummies for testing purpose)

Reply:
Done with suppliers.